package.json

```diff
@@ -25,7 +25,7 @@
     "core-object": "^1.1.0",
     "ember-cli": "0.2.7",
     "ember-cli-app-version": "0.3.3",
-    "ember-cli-aptible-shared": "0.0.24",
+    "ember-cli-aptible-shared": "0.0.25",
```

Comment:
Is this something that is easy to address? Is this something you're working on or should I take a look at it?